src/rgw/CMakeLists.txt

```diff
@@ -254,6 +254,19 @@ if(WITH_RADOSGW_ARROW_FLIGHT)
   list(APPEND librgw_common_srcs rgw_flight.cc rgw_flight_frontend.cc)
 endif(WITH_RADOSGW_ARROW_FLIGHT)
 
+file(READ rgw_redis_scripts.lua RGW_REDIS_LUA_FUNCTIONS)
+
+## FIX ME: Permission error in cmake when trying to write to LUA_INCLUDE_DIR
```

Comment:
no need to write to the lua include dir. this is the system directory holding lua headers